app/templates/content/cookies.html

```diff
@@ -86,6 +86,12 @@ <h3>Universal Analytics</h3>
                 </td>
                 <td>10 minutes</td>
             </tr>
+            <tr>
+                <td>_gat_govuk_shared</td>
+                <td>Used to manage the rate at which page view requests are made
+                </td>
+                <td>10 minutes</td>
```

Comment:
Was this meant to be included? The description copies the one from the row above...

Reply:
Good catch, might have been a rebase thing, or a sneaky left over